kroxylicious-filters/pom.xml

```diff
@@ -16,15 +16,16 @@
         <version>0.4.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>kroxylicious-additional-filters</artifactId>
+    <groupId>io.kroxylicious.filter</groupId>
+    <artifactId>kroxylicious-filter-parent</artifactId>
```

Comment:
Does this need to become a parent pom? I like the simplicity of it being a simple aggregator. 

What does adding an additional layer in the pom inheritance hierarchy buy us?

Reply:
It was already a parent pom in that the MT and schema validation poms named it as their parent. I just changed the artifactId.

Reply:
Oh my bad. I'd miss read the diff.

Let's not change it here, but I'm not convinced (even if it was my change originally).

Reply:
Ohh now I remember why I put the intermediate parent in: to make it easier to extract the modules as a separate project!

Reply:
I thought it was (also) so that the `maven-enforcer-plugin` only had to be declared once for all filters?